doc/whats_new.rst

```diff
@@ -22,29 +22,26 @@ Current
 Changelog
 ~~~~~~~~~
 
+- Adding :class:`mne_connectivity.TemporalConnectivity`, :class:`mne_connectivity.SpectralConnectivity` and :class:`mne_connectivity.SpectroTemporalConnectivity` as a data structure to hold connectivity data, by `Adam Li`_ (:gh:`6`)
+- Adding :class:`mne_connectivity.EpochTemporalConnectivity`, :class:`mne_connectivity.EpochSpectralConnectivity` and :class:`mne_connectivity.EpochSpectroTemporalConnectivity` as a data structure to hold connectivity data over Epochs, by `Adam Li`_ (:gh:`6`)
+
 Bug
 ~~~
 
 API
 ~~~
 
-Changes when mne-connectivity was part of MNE-Python
-----------------------------------------------------
-
-Changelog
-~~~~~~~~~
-
--
+- :func:`envelope_correlation`, :func:`spectral_connectivity`, and :func:`phase_slope_index` all return ``Connectivity`` containers now, by `Adam Li`_ (:gh:`6`)
+- Added ``xarray`` as a dependency where all connectivity containers are now underlying xarrays, by `Adam Li`_ (:gh:`6`)
 
-Bug
-~~~
 
--
-
-API
-~~~
+Changes when mne-connectivity was part of MNE-Python
+----------------------------------------------------
 
--
+In July, 2021, ``mne.connectivity`` submodule was ported over from the MNE-Python 
+repo into this repository, ``mne-connectivity``. Starting v0.24 of MNE-Python, that sub-module 
+will be deprecated and development will move over into this repository. Starting v0.25 of MNE-Python,
+that sub-module will completely be removed.
```

Comment:
I would consider changing "sub-module" to `mne.connectivity` in the last two sentences above, just to be very explicit.